examples/api/lib/widgets/interactive_viewer/interactive_viewer.builder.0.dart

```diff
@@ -111,23 +70,18 @@ class _IVBuilderExampleState extends State<_IVBuilderExample> {
       child: LayoutBuilder(
         builder: (BuildContext context, BoxConstraints constraints) {
           return InteractiveViewer.builder(
-            alignPanAxis: true,
-            scaleEnabled: false,
+            boundaryMargin: const EdgeInsets.all(double.infinity),
```

Comment:
I still don't fully understand the interaction between `boundaryMargin` and pan, except that it seems to be required for builder.

Reply:
Making it infinite is the right move here 👍